sql/modules/1099_reports.sql

```diff
@@ -116,8 +132,36 @@ RETURNS SETOF tax_form_report_item AS $BODY$
                 JOIN entity ON (entity.id = entity_credit_account.entity_id)
                 JOIN company ON (entity.id = company.entity_id)
                 JOIN country_tax_form ON (entity_credit_account.taxform_id = country_tax_form.id)
-               WHERE country_tax_form.id = in_tax_form_id
-             GROUP BY legal_name, meta_number, company.tax_id, company.sales_tax_id, company.entity_id, entity_credit_account.entity_class, entity.control_code, entity_credit_account.id
+  LEFT JOIN LATERAL (
+    SELECT * FROM (
+      select * from (
+      select entity_credit_account.entity_id, l.*
+        from eca_to_location eca2l
+               join location_class lc on lc.id = eca2l.location_class
+               join location l on l.id = eca2l.location_id
+       where eca2l.credit_id = entity_credit_account.id -- this is the LATERAL!
+         and lc.authoritative
+       order by lc.id, l.created
+      ) y
+       union all
+      select * from (
+      select entity.id, l.*
```

Comment:
@ylavoie, should this have been `entity_credit_account.id` so that there is at most 1 record in the right side of the join for each record on the left?